server/sv_events.lua

```diff
@@ -53,40 +55,38 @@ RegisterNetEvent('NGWD:storeVehicle', function(vehicle, garageName, plate, model
     end
     if identifier then
         if plate ~= nil and modelHash ~= nil then
-            MySQL.Async.fetchAll('SELECT * FROM ngwd_vehicles WHERE (modelHash, plate) = (@modelHash, @plate)', {
-                ['@modelHash']  = modelHash,
-                ['@plate']      = plate,
+            execute('SELECT * FROM ngwd_vehicles WHERE (modelHash, plate) = (@modelHash, @plate)', {
+                ['modelHash']  = modelHash,
+                ['plate']      = plate,
             }, function(results)
-                if results[1] then
+                if results and results[1] then
                     if results[1].owner == identifier then
-                        MySQL.Async.execute('UPDATE ngwd_vehicles SET garage = @garage, vehicleProperties = @vehicleProperties, vehicleCondition = @vehicleCondition, vehicleMods = @vehicleMods WHERE plate = @plate', { 
-                            ['@owner']                  = identifier, 
-                            ['@modelHash']              = modelHash,
-                            ['@localizedName']          = localizedName,
-                            ['@plate']                  = plate, 
-                            ['@garage']                 = garageName,
-                            ['@vehicleProperties']      = json.encode(vehicleProperties),
-                            ['@vehicleCondition']       = json.encode(vehicleCondition),
-                            ['@vehicleMods']            = json.encode(vehicleMods)
-                        }, function(affectedRows)
-                        end)             
+                        execute('UPDATE ngwd_vehicles SET garage = @garage, vehicleProperties = @vehicleProperties, vehicleCondition = @vehicleCondition, vehicleMods = @vehicleMods WHERE plate = @plate', { 
+                            ['owner']                  = identifier, 
+                            ['modelHash']              = modelHash,
+                            ['localizedName']          = localizedName,
+                            ['plate']                  = plate, 
+                            ['garage']                 = garageName,
+                            ['vehicleProperties']      = json.encode(vehicleProperties),
+                            ['vehicleCondition']       = json.encode(vehicleCondition),
+                            ['vehicleMods']            = json.encode(vehicleMods)
+                        }, function() end)             
```

Comment:
Presuming @RockySouthpaw doesn't need to do anything with affected rows, we can simply omit the cb

Reply:
I will fix this as well as the `['key']` format like you requested in Discord

Reply:
I'll make this change, as well as fixing the `['key']` format as requested tonight